hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/OMClientRequest.java

```diff
@@ -475,14 +475,17 @@ private String exceptionErrorMessage(Exception ex) {
    * @param auditLogger
    * @param auditMessage
    */
-  protected void auditLog(AuditLogger auditLogger, AuditMessage auditMessage) {
+  public void auditLog(AuditLogger auditLogger, AuditMessage auditMessage) {
     auditLogger.logWrite(auditMessage);
   }
 
   @Override
   public AuditMessage buildAuditMessage(AuditAction op,
       Map< String, String > auditMap, Throwable throwable,
-      OzoneManagerProtocolProtos.UserInfo userInfo) {
+      OzoneManagerProtocolProtos.UserInfo userInfo, TermIndex termIndex) {
+    if (null != auditMap) {
+      auditMap.put("Transaction", "" + termIndex.getIndex());
+    }
```

Comment:
This method is for building the audit message based on the input map.  `termIndex` should not be added as a parameter.  It should be added to the map by the caller.

Reply:
Chances of missing adding termIndex at every caller is high, so its done at common place. This is to avoid coding mistake, as this should be mandatory parameter for logging, and required for logging. Other parameter can be optional and based on specific request type.